pkg/client/request.go

```diff
@@ -79,6 +79,14 @@ func (r *RequestConstructionError) Error() string {
 	return fmt.Sprintf("request construction error: '%v'", r.Err)
 }
 
+type PatchType string
```

Comment:
I went with the pkg/api/types.go approach. @mikedanese would appreciate your eyes on the way I factored it out.